lib/src/api/trace/span_id.dart

```diff
@@ -20,20 +20,20 @@ class SpanId {
     id = id.padLeft(api.SpanId.sizeBits, '0');
 
     for (var i = 0; i < id.length; i += 2) {
-      _id.add(int.parse('${id[i]}${id[i + 1]}', radix: 16));
+      _id!.add(int.parse('${id[i]}${id[i + 1]}', radix: 16));
     }
   }
   SpanId.invalid() : this(List<int>.filled(sizeBytes, 0));
   SpanId.root() : this([]);
 
   /// Retrieve this SpanId as a list of byte values.
-  List<int> get() => _id;
+  List<int>? get() => _id;
 
   /// Whether this ID represents a valid [api.Span].
-  bool get isValid => _id.isEmpty || !_id.every((i) => i == 0);
+  bool get isValid => _id!.isEmpty || !_id!.every((i) => i == 0);
```

Comment:
Hi @tatashidayat,

I think it should be isNotEmpty.
```suggestion
  bool get isValid => _id!.isNotEmpty || !_id!.every((i) => i == 0);
```

Reply:
Actually, I tried to detect if it is a root span or not but it seems there is no good way.
I think I will create an extension to solve the issue then.